src/app/portlets/dot-templates/dot-template/dot-template.component.ts

```diff
@@ -0,0 +1,110 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { DotRouterService } from '@services/dot-router/dot-router.service';
+import { DotTemplatesService } from '@services/dot-templates/dot-templates.service';
+import { DotContainer } from '@shared/models/container/dot-container.model';
+import { Subject } from 'rxjs';
+import { take, takeUntil } from 'rxjs/operators';
+import * as _ from 'lodash';
+import { DotTemplate } from '@portlets/dot-edit-page/shared/models';
+
+@Component({
+    selector: 'dot-template-test',
+    templateUrl: './dot-template.component.html',
+    styleUrls: ['./dot-template.scss']
+})
+export class DotTemplateComponent implements OnInit, OnDestroy {
+    group: FormGroup;
+    editor: any;
+    isDisabled = true;
+    private destroy$: Subject<boolean> = new Subject<boolean>();
+    private originalValue: DotTemplate;
+
+    constructor(
+        private fb: FormBuilder,
+        private dotTemplateService: DotTemplatesService,
+        private dotRouterService: DotRouterService
+    ) {}
+
+    ngOnInit(): void {
+        this.group = this.fb.group({
+            title: ['', Validators.required],
+            friendlyName: '',
+            body: ['', Validators.required]
+        });
+
+        this.initialFormValues();
```

Comment:
Function names should say what they do: https://github.com/ryanmcdermott/clean-code-javascript#function-names-should-say-what-they-do

> One of the more universal, yet simple rules is: Function names should be verbs if the function changes the state of the program, and nouns if they're used to return a certain value.
https://stackoverflow.com/questions/1991324/function-naming-conventions

```suggestion
        this.setFormOriginalValue();
```